public/stylesheets/style.css

```diff
@@ -21,3 +21,7 @@
   text-decoration: line-through;
   color: #777;
 }
+
+form .error {
```

Comment:
Doesn't bootstrap provide a class for this? I would prefer to remain internally consistent with the styling until the final styleguide is applied.